numba/ir_utils.py

```diff
@@ -1401,7 +1401,7 @@ def find_callname(func_ir, expr, typemap=None, definition_finder=get_definition)
                 if hasattr(callee_def.value, key):
                     value = getattr(callee_def.value, key)
                     break
-            if not value:
+            if not isinstance(value, str):
```

Comment:
The original check that `bool(value) == False` is lost.  Is that ok?

Reply:
Good point, whilst fairly unlikely, I think it's also needed.

Reply:
b60b35f34 resolved